### ISSUE TITLE
Make search case-insensitive

### ISSUE THREAD

**Issue by dmuensterer** [openziti/ziti-console#418]
The search in the ziti-console appears to be case sensitive.
I.e. when searching for a service/policy/config by using the search term `web`

The following service/policy/config name would be found: `access_web_80`
The following service/policy/config name would **NOT** be found: `access_Web_80`

I don't think this is expected behaviour and might miss results one would be looking for.

**Comment by dovholuknf** [comment]
funny enough - we just had a conversation about this request. thanks for entering it. _currently_ this is the expected behavior. we are likely going to change this in the future for us humans :)

**Comment by dovholuknf** [comment]
here's the ziti request which was recently closed. it'll make its way into ziti-console soon: https://github.com/openziti/storage/issues/78

**Comment by dmuensterer** [comment]
Thanks for considering, appreciate the quick response!

**Comment by dovholuknf** [comment]
@rgallettonf -- we need this issue open still, right? (close it if there's already an issue, i looked but didn't see one)

**Comment by rgallettonf** [comment]
Yep we can keep this open for tracking purposes in the console, once the API supports the insensitive search. For now though this is behaving as expected.
